+exp/ExpTest.m

```diff
@@ -340,7 +372,65 @@ function setPTB(obj)
       end
     end
     
-    
+  end
+  
+  methods (Static = true)
+    function checkForUpdate(~)
```

Comment:
This shouldn't really be a method of ExpTest at all right?  Also why not use +git/update.m?  If the code in that function isn't satisfactory perhaps it should be changed...

Reply:
this just gives the user an option to decline pulling an update, and only checks for Rigbox and Signals, not the other submodules. and right, it doesn't need to be in the ExpTest class, but because I don't think it should be called in other any situation I put it in the class. Is there a better solution?

Reply:
Giving the user the option to decline is a good idea.  +git/update.m could be updated to allow this.  Couldn't you just call `update` and check all modules?  It doesn't cost much time and it's potentially important to do so as often updating alyx-matlab and not Rigbox or visa-versa causes problems.  In any case we should be pushing the latest submodule commits anyway so you shouldn't need to individually check the submodules.  My point, I suppose, is that the function of `checkForUpdate` is practically the same as `update`.  It would be better to have just one function for updating the repo, which can be called by anyone or thing that wishes to run updates.  That way you reduce code and potential errors, plus you only have to make changes to one file.

Reply:
ok, I can make the necessary changes to 'git.update' and write a test function for it in the future and call it from 'exp.ExpTest' at that time